src/test/java/core/basesyntax/service/impl/ParserImplTest.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.Parser;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ParserImplTest {
+    private static Parser parserService;
+    private static List<String> parserList;
+
+    @BeforeClass
+    public static void beforeClass() {
```

Comment:
```suggestion
    public static void setUp() {
```
change it everywhere